scripts/checkers.py

```diff
@@ -76,7 +76,7 @@ def init_pos(self) -> None:
                     self.pos_pieces[x + (y2 % 2), y2] = val
 
     def move(
-        self, selected: PosType, moveto: PosType, board: BoardType
+        self, selected: PosType, moveto: PosType, board: BoardType, taken: bool, lastmove: PosType,movedfrom: PosType

```

Comment:
Sorry, I am a little confused, how would you like me to go about doing this?

Reply:
It is working fine of course, I just would recommend using the Player class to store variables to "remember" things like the last performed move (e.g., self.lastmove), but that's just my preference, If you are okay with it and makes sense to you, we can leave it as is, no problem. I just wanted to point this out

Reply:
Thanks for pointing this out, looking back on this I should have certainly done this in the first place, it may be more complex to try alter this now but I shall have a look at this

Reply:
Ok, maybe keep it in mind for next improvements 👍 for you this is totally ok for me